fedimint-cli/src/lib.rs

```diff
@@ -793,23 +793,19 @@ impl FedimintCli {
                 Ok(CliOutput::UntypedApiOutput { value: response })
             }
             Command::Dev(DevCmd::WaitBlockCount { count: target }) => {
-                task::timeout(Duration::from_secs(60), async move {
-                    let client = self.client_open(&cli).await?;
-                    loop {
-                        let wallet = client.get_first_module::<WalletClientModule>();
-                        let count = client
-                            .api()
-                            .with_module(wallet.id)
-                            .fetch_consensus_block_count()
-                            .await?;
-                        if count >= target {
-                            break Ok(CliOutput::WaitBlockCount { reached: count });
-                        }
-                        task::sleep(Duration::from_millis(100)).await;
+                let client = self.client_open(&cli).await?;
+                loop {
+                    let wallet = client.get_first_module::<WalletClientModule>();
+                    let count = client
+                        .api()
+                        .with_module(wallet.id)
+                        .fetch_consensus_block_count()
+                        .await?;
+                    if count >= target {
+                        break Ok(CliOutput::WaitBlockCount { reached: count });
                     }
-                })
-                .await
-                .map_err_cli_msg("reached timeout")?
+                    task::sleep(Duration::from_millis(100)).await;
```

Comment:
We should log something on each failure, just to let user know what's going on and that it didn't just hang.

Reply:
That will actually lead to a log every 60s with current server-side timeouts.